lightning/src/ln/msgs.rs

```diff
@@ -543,7 +543,12 @@ pub struct UnsignedChannelUpdate {
 	pub timestamp: u32,
 	/// Channel flags
 	pub flags: u8,
-	/// The number of blocks to subtract from incoming HTLC cltv_expiry values
+	/// The number of blocks such that if:
+	/// `htlc.cltv_expiry - current_block_height <= cltv_expiry_delta`
+	/// then we need to fail the HTLC backwards. When forwarding an HTLC, cltv_expiry_delta is used to
+	/// calculate the outgoing HTLC's cltv_expiry value -- so, if an incoming HTLC comes in with a
```

Comment:
This is a big confusing, we never calculate cltv_expiry value, its specified for us in the route, we just require that the difference between the inbound and outbound legs be at least X.

Reply:
Same goes for the docs in CounterpartyForwardingInfo::cltv_expiry_delta - we don't add the value, we (well, really our counterparty) just use it as a check.

Reply:
Ah, that's a good distinction. Let me know if the updated comment looks good and I'll squash it in.

Reply:
I think we should avoid the word "calculate in the second sentence. Also, the check that's referenced isn't the relevant one, its the `if (msg.cltv_expiry as u64) < (*outgoing_cltv_value) as u64 + CLTV_EXPIRY_DELTA as u64` one.

Reply:
Updated!

Reply:
Looks good to me!